checks.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash
+
+res_pip_list=$(pip freeze)
+missing_dependencies=()
+if [ ! "$(echo $res_pip_list | grep black)" ] ; then
+  missing_dependencies+=(black)
+fi
+if [ ! "$(echo $res_pip_list | grep flake8)" ] ; then
+  missing_dependencies+=(flake8)
+fi
+if [ ! "$(echo $res_pip_list | grep isort)" ] ; then
+  missing_dependencies+=(isort)
+fi
+if [ ! "$(echo $res_pip_list | grep mypy)" ] ; then
+  missing_dependencies+=(mypy)
+fi
+if [ ! ${#missing_dependencies[@]} -eq 0 ]; then
+  echo "The following dependencies are missed:" "${missing_dependencies[@]}"
+  read -p "Would you like to install the missing dependencies? (y/N): " yn
+  case "$yn" in [yY]*) ;; *) echo "abort." ; exit ;; esac
+  pip install "${missing_dependencies[@]}"
+fi
+
+update=0
+while getopts "u" OPT
+do
+  case $OPT in
+    u) update=1
+       ;;
+    *) ;;
+  esac
+done
+
+res_black=$(black . --check 2>&1)
```

Comment:
This could be future work but making `.` a parameter would be useful if the user has local untracked files that don't comply.

Reply:
I tracked the files inside `optuna/{examples, optuna, tests}`. What do you think?

Reply:
I think that's fine as a first approach, thanks.

Reply:
Do you we should do that for flake8 and mypy as well?

Reply:
Thanks for the review. I also think we should do that for flake8 and mypy as well.

Reply:
Fixed.